hoomd/md/minimize/fire.py

```diff
@@ -0,0 +1,206 @@
+# coding: utf-8
+
+# Copyright (c) 2009-2021 The Regents of the University of Michigan
+# This file is part of the HOOMD-blue project, released under the BSD 3-Clause
+# License.
+
+# Maintainer: joaander / All Developers are free to add commands for new
+# features
+
+"""Energy minimizer."""
+
+import hoomd
+
+from hoomd.data.parameterdicts import ParameterDict
+from hoomd.data import syncedlist
+from hoomd.data.typeconverter import OnlyFrom, OnlyTypes
+from hoomd.logging import log
+from hoomd.md import _md
+from hoomd.md.integrate import _DynamicIntegrator, _preprocess_aniso
+
+
+def _raise_if_nonpositive(value):
+    if value <= 0:
+        raise ValueError("This value cannot be negative")
+    return value
+
+
+class FIRE(_DynamicIntegrator):
+    """Energy Minimizer (FIRE).
+
+    Args:
```

Comment:
As written, this section describes the `Attributes:`, not the arguments to `__init__`. After making the corrections below, rename this to `Attributes:` and move it to the end of the docstring. Then add an `Args:` section that describes the arguments to `__init__`, including  forces, constraints, methods, and rigid. You can probably copy the documentation from md.Integrator. 

Most HOOMD operations make all simple parameters arguments to `__init__` and give them default argument values when they have reasonable defaults. Unfortunately, this means some duplication in the docstring, but it allows users to explicitly see the defaults and set the parameters in a one line instantiation.

Is there a particular reason we force users to accept only the defaults on construction here? If not, add all the parameters to `__init__`.